src/main/java/pl/arcadeit/forex/stock/service/StockServiceApi.java

```diff
@@ -0,0 +1,22 @@
+package pl.arcadeit.forex.stock.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pl.arcadeit.forex.stock.dao.StockDao;
+import pl.arcadeit.forex.stock.entity.Stock;
+
+@Service
+public class StockServiceApi implements StockService {
+
+    private final StockDao stockDao;
+
+    @Autowired
+    public StockServiceApi(StockDao stockDao) {
```

Comment:
1. @Autowired annotation is unnecesary on constructor
2. Consider adding Primary or Qualifier to StockDao, otherwise application won't run